app/src/pages/playground/ModelConfigButton.tsx

```diff
@@ -331,11 +331,9 @@ function ModelConfigDialogContent(props: ModelConfigDialogContentProps) {
               onChange={onModelNameChange}
             />
           )}
-          {instance.model.modelName ? (
+          <Suspense>
```

Comment:
do we wanna put a loader here or just null? didn't see in the video how this looks, if you go with 2 below then doesn't matter